src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -218,7 +220,7 @@ var DOMPropertyOperations = {
    * @param {DOMElement} node
    * @param {string} name
    */
-  deleteValueForProperty: function(node, name) {
+  deleteValueForProperty: function(node, name, isCustomElement) {
```

Comment:
See my comments above for `setValueForProperty` -- I'm changing this function for much the same reason.